src/brc/api/viewsets.py

```diff
@@ -14,6 +15,7 @@ class BesluitViewSet(mixins.CreateModelMixin,
                      mixins.ListModelMixin,
                      mixins.RetrieveModelMixin,
                      mixins.UpdateModelMixin,
+                     mixins.DestroyModelMixin,
```

Comment:
this effectively turns it into a simple `(viewsets.ModelViewSet):`

Reply:
explicit is better than implicit no? but you are right, i just noticed it was done like this in other places.

Reply:
Mostly because there were no pre-baked base classes :)